library/bignum.c

```diff
@@ -1877,19 +1877,31 @@ int mbedtls_mpi_mod_int( mbedtls_mpi_uint *r, const mbedtls_mpi *A, mbedtls_mpi_
 
 /*
  * Fast Montgomery initialization (thanks to Tom St Denis)
```

Comment:
While you're at it, please document that `mm` must have room for at least one limb, and that this function only sets the lowest-order limb of `mm`.